### ISSUE TITLE
Retrain `bert_tiny_uncased_en_sst2` BertClassifier to reflect the dropout change

### ISSUE THREAD

**Issue by chenmoneygithub** [keras-team/keras-hub#542]
We added dropout layer to `keras_nlp.models.BertClassifier`, so we need to update the presets accordingly.

**Comment by chenmoneygithub** [comment]
@jbischof Jon I believe you still have the training script?

**Comment by jbischof** [comment]
Yes it's in our repo ([link](https://github.com/keras-team/keras-nlp/blob/master/tools/checkpoint_training/bert_tiny_uncased_en_sst2_training.ipynb)).

My goal was having a task preset for primary API development and tutorial writing. It's OK if these models get better over time.

**Comment by susnato** [comment]
Hi @chenmoneygithub I would like to solve this issue. Could you please guide me little bit? As I understood I need to train `BertClassifier` 2 times with different `dropout_probs` to reflect how this change is changing the final trained training...is it right? 

cc : @jbischof

**Comment by chenmoneygithub** [comment]
@susnato Thanks for your interest!

To clarify - we finetuned BERT earlier on SST2 to make `bert_tiny_uncased_en_sst2`, however, dating back to that time our `BertClassifier` did not have the dropout layer. So what we need to do now is to finetune `BertClassifier` again on SST2. Your work will include:
1) Write a colab that finetunes `keras_nlp.models.BertClassifier` on SST2.
2) report the evaluation score on validation set. 
3) share the colab with us by opening a PR to 

Then we will run your colab to generate the checkpoint and upload to our Google cloud storage. Since no code will be checked in, we will explicitly credit you in the code and our documentation on keras-io.

**Comment by susnato** [comment]
@chenmoneygithub Thanks for replying! For how many epochs should I train the model?

**Comment by jayam30** [comment]
Hey i would like to take this up

**Comment by susnato** [comment]
Hi @jayam30 as you can see I have already submitted the PR regarding the issue and we are working on this. 
You can choose other the issues that need immediate fix from [this list](https://github.com/keras-team/keras-nlp/projects/2)
